example/spec/parallel/syncGranule/SyncGranuleSuccessSpec2.js

```diff
@@ -0,0 +1,466 @@
+const fs = require('fs');
```

Comment:
I think this was accidentally committed. Can you remove it? Sorry you're working on multiple things at once and context switching 😓

Reply:
Ah I see you deleted this in this commit https://github.com/nasa/cumulus/pull/3208/commits/c6ab561e372a0b74bfd90dc1b9c6b4013318e7a0